syntax = "proto3";

package evaluation;

service Environment{
  rpc reset(Package) returns (Package) {}
  rpc get_action_space(Package) returns (Package) {}
  rpc act_on_environment(Package) returns (Package) {}
  rpc get_output_keys(Package) returns (Package) {}
  rpc set_output_keys(Package) returns (Package) {}
}

message Package{
  bytes SerializedEntity = 1;
}
